src/commands/edge_app.rs

```diff
@@ -1649,31 +1617,25 @@ mod tests {
 
     #[test]
     fn test_upload_should_send_correct_requests() {
-        let manifest = EdgeAppManifest {
-            app_id: Some("01H2QZ6Z8WXWNDC0KQ198XCZEW".to_string()),
-            entrypoint: None,
-            user_version: "1".to_string(),
-            description: "asdf".to_string(),
-            icon: "asdf".to_string(),
-            author: "asdf".to_string(),
-            homepage_url: "asdfasdf".to_string(),
-            settings: vec![
-                Setting {
-                    type_: "string".to_string(),
-                    title: "asetting".to_string(),
-                    optional: false,
-                    default_value: "".to_string(),
-                    help_text: "".to_string(),
-                },
-                Setting {
-                    type_: "string".to_string(),
-                    title: "nsetting".to_string(),
-                    optional: false,
-                    default_value: "".to_string(),
-                    help_text: "".to_string(),
-                },
-            ],
-        };
+        let mut manifest = create_edge_app_manifest_for_test(vec![
```

Comment:
Entrypoint was None for this test before, though it is replaced with Some.

Reply:
Good point, I'm fixing it